core/src/main/java/org/commonjava/maven/ext/manip/state/DependencyState.java

```diff
@@ -96,7 +101,13 @@ public DependencyState( final Properties userProps )
         ignoreSuffix = Boolean.valueOf( userProps.getProperty( STRICT_ALIGNMENT_IGNORE_SUFFIX, "false" ) );
         failOnStrictViolation = Boolean.valueOf( userProps.getProperty( STRICT_VIOLATION_FAILS, "false" ) );
         remoteBOMdepMgmt = IdUtils.parseGAVs( userProps.getProperty( DEPENDENCY_MANAGEMENT_POM_PROPERTY ) );
+
         dependencyExclusions = PropertiesUtils.getPropertiesByPrefix( userProps, DEPENDENCY_EXCLUSION_PREFIX );
+        // If dependencyExclusions have not been set via exclusion prefix attempt to set it via the override alias prefix.
+        if ( dependencyExclusions.isEmpty())
+        {
+            dependencyExclusions = PropertiesUtils.getPropertiesByPrefix( userProps, DEPENDENCY_OVERRIDE_PREFIX );
```

Comment:
Thats a good question. Should it be additive, should one take precedence, should only one be allowed etc. Using `putAll` is one option to combine but is does mean we need to decide which should take precedence.